### PR TITLE
Classic Theme Helper: Require Content Options from the package

### PR DESCRIPTION
**Pull request by coder-karen**
Fixes https://github.com/Automattic/vulcan/issues/466
Fixes https://github.com/Automattic/vulcan/issues/468
Fixes https://github.com/Automattic/vulcan/issues/467

Note: The Jetpack Mu WPCOM plugin will need to be updated before Jetpack (so a wpcomsh release before Jetpack on WoA), otherwise Content Options will not display until it is (on WoA sites).

## Proposed changes:

* This PR ensures we are requiring the newly moved Content Options code from the Classic Theme Helper package, in the Jetpack, Jetpack Mu WPcom plugins and Classic Theme Helper plugin.
* In the process, I've also removed all jQuery from customizer.js and replaced with vanilla JS
* I've also made use of the Assets package to register and enqueue the customizer.js (and it's min version) script.
* Lastly, I've fixed one previously Phan-suppressed issue

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [x] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

https://github.com/Automattic/vulcan/issues/466

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

**Self-hosted:**

* With this PR applied on a test site, ensure you have a theme active that supports Content Options (Dara is a good example).
* In the customizer, select 'Content Options'. 
* Experiment with changing blog display, and all other settings (you may need to add featured images / categories to some posts to be able to test everything).

**WoA**

* Test as with self-hosted, but ensure you apply the PR to both Jetpack and the WordPress.com Features plugins. You can test with just WordPress.com Features having the PR applied too - this won't work though with just the PR applied on Jetpack (a wpcomsh release will be needed before a Jetpack on Atomic release after this PR is merged).

**Simple**

* Test as with Self-hosted using the commands in the generated comment below. Note that Content Options won't work with only the Jetpack branch applied, however this is fine as when deployed on WPcom both Jetpack Mu WPcom and Jetpack are updated.

**Classic Theme Helper plugin**

* If testing locally, with the PR applied make sure you have installed and built the Classic Helper Theme plugin: `jetpack install plugins/classic-theme-helper-plugin`, `jetpack build plugins/classic-theme-helper-plugin`, then activate the plugin.
* Also if testing locally, remove `"automattic/jetpack-classic-theme-helper": "@dev"`, from the Jetpack plugin `composer.json` file. Then run `tools/fixup-project-versions.sh`, and then `jetpack install plugins/jetpack` and` jetpack build plugins/jetpack`.
* If testing using the Beta tester plugin, activate the branch for the Classic Theme Helper plugin.
* Comment out `theme-tools/content-options.php` from `projects/plugins/jetpack/modules/module-extras.php` (and from `jetpack-dev/modules/module-extras` via SFTP / SSH if using a test site and the Beta tester plugin)
  * If testing using the Beta tester plugin, delete the `jetpack-classic-theme-helper` package folder from `jetpack-dev/jetpack_vendor/automattic`.
* Follow the steps under 'Self-hosted' to test Content Options.

In all cases, there should be no deprecation notices or errors in logs.

### CODE REVIEW

**Review by sergeymitr — APPROVED** [review]
Tests well 👍

**Comment by coder-karen** [comment]
Thanks @sergeymitr ! I'll not merge this until after the Jetpack on WoA release is cut next week, but in the meantime will leave it open for any more reviews, primarily as the customizer.js file has been completely rewritten.

**Pull request merged**
